### ISSUE TITLE
Index page of NVDA Community portal should have main content starting with a heading

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5369]
**Reported by bhavyashah on 2015-09-18 10:19**
_This is a very trivial issue._
Please visit http://community.nvda-project.org/wiki/TitleIndex
To streamline it with the rest of the community portal and for quicker navigation, there should be a heading on the page labelled 'Index' and then should appear the content Index by Title | Index by Date...

**Comment by leonardder** [comment]
Closing as invalid, since NVDA community website moved away from trac.
